.github/workflows/flowzone.yml

```diff
@@ -603,38 +607,29 @@ jobs:
 
     defaults:
       run:
-        working-directory: ${{ inputs.working_directory }}
+        working-directory: .
         shell: bash --noprofile --norc -eo pipefail -x {0}
 
     steps:
-      - name: Download source artifact
-        uses: actions/download-artifact@v3
+      # https://github.com/dawidd6/action-download-artifact
+      # TODO: what if this is a tag event and PR artifacts do not exist?
+      - name: Download artifacts from PR
+        uses: dawidd6/action-download-artifact@v2
         with:
-          name: source-${{ github.event.pull_request.head.sha }}
-
-      - name: Extract source artifact
-        working-directory: .
-        run: tar -xvf source.tar
+          github_token: ${{ secrets.FLOWZONE_TOKEN }}
+          commit: ${{ github.event.pull_request.head.sha }}
+          path: /tmp
+          workflow_conclusion: success
 
       - name: Login to registry
         run: |
           echo '//${{ env.NPM_REGISTRY }}/:_authToken=${{ secrets.NPM_TOKEN }}' > ~/.npmrc
           npm whoami
 
-      - name: Install dependencies, if necessary
-        run: |
-          if grep -E '(preversion|postversion|prepare|prepack|postpack|prepublishOnly|publish)' package.json; then
-            # If a relevant publish script exists then install dependencies before publishing
-            if [ -e package-lock.json ]; then
-              npm ci
-            else
-              npm i
-            fi
-          fi
-
       - name: Publish final release
         run: |
-          npm --loglevel=verbose --logs-max=0 publish --tag "latest"
+          pack="$(ls /tmp/npm-*/*.tgz | tail -n1)"
```

Comment:
Is this guaranteed to find the latest? I don't know if it'll hit issues with using lexical rather than numerical sorting?

Reply:
We can add a `| sort -V` to to try and sort more reliably? I didn't want to spend too much time on this part.